packages/firestore/test/unit/specs/bundle_spec.test.ts

```diff
@@ -270,13 +292,93 @@ describeSpec('Bundles:', ['no-ios', 'no-android'], () => {
     );
   });
 
+  specTest('Bundles query can be resumed from same query.', [], () => {
+    const query1 = query('collection');
+    const docA = doc('collection/a', 100, { key: 'a' });
+    const bundleString1 = bundleWithDocumentAndQuery(
+      {
+        key: docA.key,
+        readTime: 500,
+        createTime: 250,
+        updateTime: 500,
+        content: { value: 'b' }
+      },
+      { name: 'bundled-query', readTime: 400, query: query1 }
+    );
+
+    return spec()
+      .loadBundle(bundleString1)
+      .userListens(query1, 400)
```

Comment:
Please add a comment for non-descriptive arguments. I was just writing a comment on how the target here should have a read time when I realized that this is already provided.